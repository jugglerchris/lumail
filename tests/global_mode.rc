mode = global_mode()
io.write(mode.."\n")
global_mode('text')
mode = global_mode()
io.write(mode.."\n")
