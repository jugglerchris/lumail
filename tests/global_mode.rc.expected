maildir
text
Exit: 0
